lib/pqos.h

```diff
@@ -366,6 +369,7 @@ struct pqos_coreinfo {
         unsigned l2_id;    /**< L2 cluster id */
         unsigned l3cat_id; /**< L3 CAT classes id */
         unsigned mba_id;   /**< MBA id */
+        unsigned smba_id;  /**< SMBA id */
```

Comment:
Add this smba_id at end of the structure. It breaks ABI compatibility

Reply:
Sure. Will do.